guides/git-submodules.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+sublayout: guide
+title: Git Triggers & Submodules
+---
+
+[Git submodules](https://git-scm.com/docs/git-submodule) are a popular way for one git repository to _vendor_ dependencies on other git repositories.
+
+Quay.io supports the ability to build repositories that use submodules, however with some manual setup is required.
+When Quay.io creates a Git Trigger, it creates an SSH public key that it adds to the repository's deploy key section.
```

Comment:
Git Triggers make use of SSH public keys to gain access to the source repository. Quay.io will create and add automatically when the trigger is setup.